styles/style.css

```diff
@@ -0,0 +1,130 @@
+body {
+    background-color: #eceef1;
+}
+
+.top {
+    height: 70px;
+}
+
+.top-text {
+    float: left;
+    font-size: 0.9em;
+    line-height: 0.7;
+    text-align: right;
+    width: 90%;
+}
+
+.signin-btn {
+    display: inline-block;
+    margin-left: 8px;
+    margin-top: 8px;
+}
+
+.signin-btn button {
+    background-color: #0176c5;
+    border-radius: 3px;
+    color: white;
+    cursor: pointer;
+    font-size: 1.5em;
+    padding: 5px;
+}
+
+header {
+    font-family: 'Lobster', cursive;
+    line-height: .7;
+    position: relative;
+    text-align: center;
+}
+
+h3 {
+    display: inline;
+}
+
+h3 a {
+    color: black;
+}
+
+.form-box {
+    margin: 0 auto;
+    width: 450px;
+}
+
+form {
+    background-color: white;
+    border: 1px solid black;
+    margin-top: 20px;
+}
+
+form div {
+    margin: 1em;
+}
+
+.form-title {
+    line-height: 0.7;
+    text-align: center;
+}
+
+.form-title p {
+    line-height: 1;
+}
+
+input {
+    font-size: 15px;
+    height: 30px;
+    margin: 0 auto;
+    width: 98%;
+}
+
+.form-label {
+    font-size: 1.1em;
+    margin-top: 30px;
+    padding: 0;
+}
+
+.form-label p{
+    font-size: 13px;
+    margin-top: 0;
+}
+
+.form-disclaimer {
+    font-style: italic;
+    text-align: center;
+}
+
+form button {
+    background-color: #0176c5;
+    color: white;
+    cursor: pointer;
+    font-size: 14px;
+    margin: 0 auto;
+    padding-bottom: 8px;
+    padding-top: 8px;
+    width: 100%;
+}
+
+.form-footer {
+    background-color: #eceef1;
+    margin: 0;
+}
+
+.form-footer p {
+    font-size: 13px;
+    margin: 0;
+    padding: 15px;
+    text-align: center;
+}
+
+a {
```

Comment:
Maybe, for structure, you can put this on closer to the beginning of file way you implement general styles.